<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmpReport</class>
 <widget class="QDialog" name="EmpReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employee Report</string>
  </property>
  <widget class="QPushButton" name="buttonHour">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Hours Worked</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonProject">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Hours Worked for a Project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonVacation">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Vacation/Sick Leave</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonYear">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Pay year-to-date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>461</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBegin">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>begin date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>end date:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEnd">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
